SillyBusinessInc/SillyBusinessGame
# Feat:#106 tail upgrades

## Purpose of this PR:
Functionality of a tailupgrade added, Wafflequake and Double Tap, see miro for description

## How to test:
Add a action to a object and add either wafflequake or doubletap. Make sure to add interactable script to it, it should work from then

### links: 
[Ticket](https://github.com/SillyBusinessInc/SillyBusinessGame/issues/106)
